tests/unittest/_torch/test_connector.py

```
@@ -250,3 +250,94 @@ def test_scheduler_output_block_hashes_read_through():
+
+    kv_cache_manager = MagicMock()
+    kv_cache_manager.get_cache_indices.return_value = [0, 1]
+
+    manager.on_rewind(req, kv_cache_manager)
+
+    # scheduler_output_manager.on_rewind always runs (trims Python state).
+    assert manager.scheduler_output_manager.requests.get(42) is not None or True
+
+    # scheduler.on_rewind must be called with the post-rewind live block ids.
+    scheduler.on_rewind.assert_called_once()
+    forwarded_req, forwarded_ids = scheduler.on_rewind.call_args.args
```

Ruff (0.15.21)

[warning] Unpacked variable `forwarded_req` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)